backend/app/views/spree/admin/cancellations/index.html.erb

```diff
@@ -34,7 +34,15 @@
             <% end %>
           </td>
           <td class="inventory-unit-quantity align-center">1</td>
-          <td class="inventory-unit-state align-center"><%= inventory_unit.state %></td>
+          <td class="inventory-unit-state align-center">
+            <span class="pill pill-<%= inventory_unit.state %>">
+              <%= Spree.t(
+                inventory_unit.state,
+                scope: :inventory_states,
+                default: inventory_unit.state.titleize
+              ).titleize %>
```

Comment:
I like that this is now translatable, but please do not `titleize` this. This is what should be done in the translation itself, upper and lower case rules are very specific to languages.

Also, this should be a separate commit.